message_auto_subscribe_notify_own/tests/test_fake_model.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2019 Aleph Objects, Inc.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class FakeModelToTest(models.Model):
+    """A Fake model to Test."""
+
+    _name = 'fake.model'
```

Comment:
`message_auto_subscribe_notify_own.test` should be a better name. This way we can identify the table from which module comes from.

Reply:
Ok! it makes sense, changed